config/babel.config.js

```diff
@@ -26,6 +30,11 @@ module.exports = {
           transform: '@fortawesome/free-solid-svg-icons/${member}',
           skipDefaultConversion: true,
         },
+        // lodash
+        lodash: {
+          transform: 'lodash/${member}',
+          skipDefaultConversion: true,
```

Comment:
Ah, I think this is the problem. Don't we need `skipDefaultConversion: false` ?

Reply:
Yes, you are right